.travis.yml

```diff
@@ -40,10 +40,11 @@ before_deploy:
 - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH;
   else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
 - export BRANCH_CLEAN=${BRANCH//\//_}
+- export COMMIT_SHA=${TRAVIS_COMMIT:0:7}
 - mkdir deploy
 - ls target/*.jar
-- cp target/otp-middleware-*.jar deploy/
-- cp target/otp-middleware-*.jar "deploy/otp-middleware-latest-$BRANCH_CLEAN.jar"
+- cp target/otp-middleware.jar "deploy/otp-middleware-$BRANCH_CLEAN-$COMMIT_SHA.jar"
```

Comment:
Let's just use commit_sha. No need to keep the branch name there.

Reply:
Also, I think my original comment remarked that we should use `git describe`, which is not the same as commit_sha. This is fine for now though.

Reply:
I think it is better we show the branch name, that way we can set the continuous deployment process to find the files for that branch. And it is better for us for testing/troubleshooting too.